scripts/abbreviations/main.swift

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env xcrun swift -F ./Carthage/Build/Mac
```

Comment:
This is a pretty well-designed script. 👍 Do you think we should eventually port json2plist.sh to Swift for consistency? (I like json2plist.sh’s brevity, but I don’t know if that would be achievable for this abbreviation script in Bash – maybe with `PlistBuddy`? 🤷‍♂️)